cdc/sink/mysql_params_test.go

```diff
@@ -74,6 +74,7 @@ func TestGenerateDSNByParams(t *testing.T) {
 			"writeTimeout=2m",
 			"allow_auto_random_explicit_insert=1",
 			"transaction_isolation=%22READ-COMMITTED%22",
+			"tidb_placement_mode=ignore",
```

Comment:
This may failed for a real connection because ignore is a keyword. I think we can do it like `transaction_isolation` to quote the value like this: `tidb_placement_mode=%22ignore%22`